{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "cveId": "CVE-2025-27784",
    "assignerOrgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
    "state": "PUBLISHED",
    "assignerShortName": "GitHub_M",
    "dateReserved": "2025-03-06T18:06:54.461Z",
    "datePublished": "2025-03-19T20:41:32.140Z",
    "dateUpdated": "2025-03-19T20:41:32.140Z"
  },
  "containers": {
    "cna": {
      "title": "Applio allows arbitrary file read in train.py export_pth function",
      "problemTypes": [
        {
          "descriptions": [
            {
              "cweId": "CWE-200",
              "lang": "en",
              "description": "CWE-200: Exposure of Sensitive Information to an Unauthorized Actor",
              "type": "CWE"
            }
          ]
        }
      ],
      "metrics": [
        {
          "cvssV4_0": {
            "attackVector": "NETWORK",
            "attackComplexity": "LOW",
            "attackRequirements": "NONE",
            "privilegesRequired": "NONE",
            "userInteraction": "NONE",
            "vulnConfidentialityImpact": "HIGH",
            "vulnIntegrityImpact": "NONE",
            "vulnAvailabilityImpact": "NONE",
            "subConfidentialityImpact": "NONE",
            "subIntegrityImpact": "NONE",
            "subAvailabilityImpact": "NONE",
            "baseScore": 7.7,
            "baseSeverity": "HIGH",
            "vectorString": "CVSS:4.0/AV:N/AC:L/AT:N/PR:N/UI:N/VC:H/VI:N/VA:N/SC:N/SI:N/SA:N/E:P",
            "version": "4.0"
          }
        }
      ],
      "references": [
        {
          "name": "https://securitylab.github.com/advisories/GHSL-2024-341_GHSL-2024-353_Applio/",
          "tags": [
            "x_refsource_CONFIRM"
          ],
          "url": "https://securitylab.github.com/advisories/GHSL-2024-341_GHSL-2024-353_Applio/"
        },
        {
          "name": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/tabs/train/train.py#L267",
          "tags": [
            "x_refsource_MISC"
          ],
          "url": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/tabs/train/train.py#L267"
        },
        {
          "name": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/tabs/train/train.py#L801",
          "tags": [
            "x_refsource_MISC"
          ],
          "url": "https://github.com/IAHispano/Applio/blob/29b4a00e4be209f9aac51cd9ccffcc632dfb2973/tabs/train/train.py#L801"
        }
      ],
      "affected": [
        {
          "vendor": "IAHispano",
          "product": "Applio",
          "versions": [
            {
              "version": "<= 3.2.8-bugfix",
              "status": "affected"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
        "shortName": "GitHub_M",
        "dateUpdated": "2025-03-19T20:41:32.140Z"
      },
      "descriptions": [
        {
          "lang": "en",
          "value": "Applio is a voice conversion tool. Versions 3.2.8-bugfix and prior are vulnerable to arbitrary file read in train.py's `export_pth` function. This issue may lead to reading arbitrary files on the Applio server. It can also be used in conjunction with blind server-side request forgery to read files from servers on the internal network that the Applio server has access to. As of time of publication, no known patches are available."
        }
      ],
      "source": {
        "advisory": "GHSA-6gr3-4mpf-g6gx",
        "discovery": "UNKNOWN"
      }
    }
  }
}